packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -8,19 +8,18 @@
 import * as path from 'path';
 import * as vscode from 'vscode';
 import {
-  Executable,
   LanguageClientOptions,
   RevealOutputChannelOn
 } from 'vscode-languageclient';
```

Comment:
Looking at the samples I think these should also be pulled in from 'vscode-languageclient/node'